service-front/config/autoload/development.local.php.dist

```diff
@@ -0,0 +1,23 @@
+<?php
```

Comment:
In general we'll put any dev-level config in via environment config at the docker-compose level, so this can be removed.